Cargo.toml

```diff
@@ -9,13 +9,13 @@ exclude = ["/.cirrus.yml", "/.github/*"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = { version = "3.0.12", features = ["derive"] }
+clap = { version = "3.0.14", features = ["derive"] }
 env_logger = "0.9.0"
 log = "0.4.14"
 trust-dns-server = "0.20.3"
-trust-dns-proto = "0.20.3"
-trust-dns-client = "0.20.3"
-anyhow = "1.0.39"
+trust-dns-proto = "0.20.4"
+trust-dns-client = "0.20.4"
```

Comment:
Should we bump trust-dns after `aardvark` gets stable, maybe in 2-3 weeks ? `0.20.4` just got released I guess.

Edit: Not a blocker.